src/features/services/components/data-settings.tsx

```diff
@@ -14,31 +14,28 @@ import {
 import { PutValueModal } from "../components";
 import { useAccountsStore } from "features/accounts";
 import { ANON_IDENTITY } from "@liftedinit/many-js";
-// import { useDataServiceStore } from "features/services";
+import { useDataServiceStore } from "features/services";
 
 interface KVData {
   key: string;
-  value: string;
-  tag: string;
 }
 
-// eslint-disable-next-line
-function KVDataRow({ key, value, tag }: KVData) {
-  return (
-    <Tr key={key}>
-      <Td>{key}</Td>
-      <Td>{value}</Td>
-      <Td>{tag}</Td>
-    </Tr>
-  );
+function KVDataRow(keys: any) {
+  const kvlist = keys.map((key:KVData) =>
+  <Tr>
+    <Td>{key}</Td>
+    <Td></Td>
+    <Td></Td>
+  </Tr>
+  ); 
+  return kvlist;
 }
 
 export function DataSettings() {
   const account = useAccountsStore((s) => s.byId.get(s.activeId));
-  // @TODO: Fetch all the keys
-  // const keys = useDataServiceStore((s) => s.keys);
+  const keys = useDataServiceStore((s) => s.keys);
```

Comment:
I'd suggest doing what I do here instead: https://github.com/stanleyjones/gwen/pull/1

If you do it this way, you'll only get the keys and you want the values too.